### ISSUE TITLE
Any way to make D7000 a DNS server (dnsmasq?)

### ISSUE THREAD

**Issue by Dronec** [negan07/ancistrus#94]
Hi, is it possible at all to add dnsmasq to the router, so it could resolve intranet names (e.g. foo.local -> 192.168.1.2)? Or maybe there's an easy way to add records to /etc/hosts?

**Comment by negan07** [comment]
yes It could but It will break some vendor dns related vanilla support
